cli/worker.rs

```diff
@@ -170,23 +152,65 @@ impl Future for Worker {
     }
   }
 }
+#[derive(Clone)]
+pub struct MainWorker(Worker);
```

Comment:
I also think it would be helpful to have some triple slash comments here explaining this is the Isolate one gets by default when they start a script. It does not include TS compiler, but does include "Deno" global, etc.